airgun/views/common.py

```diff
@@ -56,6 +62,33 @@ class listremove(SatSecondaryTab):
         './/nav[@class="ng-scope"]/following-sibling::div')
 
 
+class LCESelectorGroup(ParametrizedView):
+    ROOT = ".//div[@path-selector='environments']"
+
+    PARAMETERS = ('lce_name',)
+
+    LAST_ENV = ".//div[contains(@class, 'path-selector')]/ul/li[last()]"
+    lce = LCESelector(
+        locator=ParametrizedLocator(
+            "./div[contains(@class, 'path-selector')]/ul"
+            "[li[normalize-space(.)='{lce_name}']]")
+    )
+
+    @classmethod
+    def all(cls, browser):
+        return [(element.text,) for element in browser.elements(cls.LAST_ENV)]
+
+    def fill(self, values=None):
+        if values in (True, False):
+            values = {self.context['lce_name']: values}
+        else:
+            values = {values: True}
```

Comment:
as per our discussion, you will read only last element, so why do we need fill then?

Reply:
Probably you haven't read my last reply. There's no point in not supporting filling or reading of all the elements, it's parametrizedview which adds extra layer (extra dict) to interface, not this feature. Interface won't become simpler and we won't win anything from that.

Reply:
Instead of
```python
{
    'name': 'some_name',
    'description': 'some_description',
    'lce': {
        'DEV': {'Library': False, 'DEV': False},
        'IT': {'Library': False, 'IT': False},
        'PROD': {'Library': False, 'PROD': False},
        [...]
    },
}
```
we'll receive
```python
{
    'name': 'some_name',
    'description': 'some_description',
    'lce': {
        'DEV': {'DEV': False},
        'IT': {'IT': False},
        'PROD': {'PROD': False},
        [...]
    },
}
```
That doesn't change anything for user, he'll still have to access them like `ak_values['lce'][env_name][env_name]`

Reply:
sure, but you mentioned things like `assert any(lce[env_name] == True for lce in ak_values['lce'].values())` which is definitely a waaaaaaaaaaay too complex for test layer. I recommend to simplify your solution as more as you can

Reply:
Well, that's in case we want to verify value of other than latest lce. If we have a complex corner case - its assertion won't be trivial as well. For common case with latest element it's just
```python
assert ak_values['lce'][env_name][env_name]
```
or 
```python
assert not ak_values['lce'][env_name][env_name]
```